### ISSUE TITLE
Transitions Between Screens when Navigating Application

### ISSUE THREAD

**Issue by deanbrenden93** [thunkable/thunkable-issues#471]
Our apps from Thunkable would look so much more sleek and professional if we had the option of implementing animated transitions to ease users into the page they just clicked instead of an uncomfortable jump cut and often blinding white screens when loading a webpage.

Keep in mind it would probably be best that a loading icon appears on the screen so that the next screen can load before the animated transition occurs.

**Comment by jaredgib** [comment]
Yes! Stuff like this https://www.youtube.com/watch?v=PvjV96CNPqM
